public/index.css

```diff
@@ -1,19 +1,177 @@
 .header {
```

Comment:
We should probably add `public/*.css` `public/*.js` `public/*.html` to the .gitignore to clean up the diff.